src/assets/icons/arrow-down.svg

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 25.0.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 20 28" style="enable-background:new 0 0 20 28;" xml:space="preserve">
+<path d="M10,27.9c-0.1,0-0.1,0-0.2-0.1l-9.7-7.8C0,20,0,19.8,0.1,19.6c0.1-0.1,0.3-0.1,0.4,0L9.7,27V0.4c0-0.2,0.1-0.3,0.3-0.3
+	c0.2,0,0.3,0.1,0.3,0.3V27l9.2-7.4c0.1-0.1,0.3-0.1,0.4,0c0.1,0.1,0.1,0.3,0,0.4l-9.7,7.8C10.1,27.9,10.1,27.9,10,27.9z"/>
+</svg>
```

Comment:
- why not using the icon component?
- afaik we don't have a svg minifier, so we need to keep it clean